src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentStageDetailsBlock/partials/PaymentItem/types.ts

```diff
@@ -0,0 +1,9 @@
+import { type Token } from '~types/graphql.ts';
+
+export interface PaymentItem extends Token {
+  amount: string;
+}
+
+export interface PaymentItemProps {
+  payouts: PaymentItem[];
```

Comment:
At some point this will need to work on the GraphQL types such as `ExpenditureSlot`, `ExpenditurePayouts`, etc. So maybe not something to change now, but to keep in mind when we get there.